rudder-agent/SOURCES/Makefile

```diff
@@ -199,10 +199,10 @@ build: get-flags build-cfengine
 # install to DESTDIR
 install: build $(INSTALL_DEPS) install-cfengine initial-promises initial-ncf rudder-agent-utilities rudder-agent.cron
 	# Directories
-	mkdir -p $(INSTALL_DIR)/etc
-	mkdir -p $(INSTALL_DIR)/bin
-	mkdir -p $(INSTALL_DIR)/share/commands
-	mkdir -p $(INSTALL_DIR)/share/lib
+	mkdir -p $(DESTDIR)/opt/rudder/etc
+	mkdir -p $(DESTDIR)/opt/rudder/bin
+	mkdir -p $(DESTDIR)/opt/rudder/share/commands
+	mkdir -p $(DESTDIR)/opt/rudder/share/lib
 	mkdir -p $(DESTDIR)/var/rudder/cfengine-community/bin
```

Comment:
The reason if this change can be seen here, mixing a little bit of INSTALL_DIR and DESTDIR everywhere make it difficult to read and understand where the final destination is